libs/shared/services/src/lib/api-routes.constant.ts

```diff
@@ -836,6 +836,7 @@ export const API_ROUTES = {
       INDEX_PROJECTS: 'api/v2/hackathons/public/index_projects', //GET
       INTERESTED_USERS: 'api/v2/hackathons/public/interested_users', //GET
       IS_MEMBER_OF_PARENT: 'api/v2/hackathons/public/is_member_of_parent', //GET
+      UPCOMING_HACKATHONS: 'api/v2/hackathons/public/all_hackathons', //GET
```

Comment:
This will return all hackathons so why is it named upcoming hackathons, In future we need this to show past hackathons too!